azkaban-exec-server/src/test/java/azkaban/execapp/FlowRunnerPipelineTest.java

```diff
@@ -67,7 +65,7 @@
  *
  * @author rpark
  */
-public class FlowRunnerPipelineTest {
+public class FlowRunnerPipelineTest extends FlowRunnerTestBase {
```

Comment:
@HappyRay has been wanting to get rid of inheriting FlowRunnerTestBase and having some static utility methods instead.